rethinkdb/tests/compose/docker-compose.yaml

```diff
@@ -0,0 +1,62 @@
+version: "3"
+
+services:
+  # 3-node RethinkDB cluster with 1 proxy node.
+
+  rethinkdb-server0:
+    tty: true  # Required otherwise RethinkDB won't output any logs.
```

Comment:
Actually none or they go to a file?

Reply:
They still go to a file, yes. I just found out the actual reason: on RethinkDB 2.4.0 without a TTY the I/O is buffered (so `docker logs` won't see them until basically the container shuts down). Linked issue: https://github.com/rethinkdb/rethinkdb/issues/6819. Bug fix seems planned for 2.4.1. I'll update the comment here with a link to that issue. :-)